posts/2024-04-03-march-progress-update.mdx

```diff
@@ -0,0 +1,104 @@
+* The research team has submitted two academic papers that are currently under review. 
+
+
+---
+
+# Community call
+
+Waku hosts a Community Call on X during the last week of each month. The objective of the call is to update the community on our progress and provide a space where questions can be put directly to the team.
+
+The Community Call took place on the 29th of March. You can [listen back to the full Space](https://twitter.com/i/spaces/1lDxLPRNqaMxm?s=20) and read the highlights below. [Aaron](https://twitter.com/vrycmfy), [Guru](https://twitter.com/hackyguru), [Vaclav,](https://twitter.com/vpavlin) and [Danish](https://twitter.com/danisharora099) from the Waku team led the call. 
+
+## ETHLatam winners: EduChain
+
+The Waku team was joined by bounty winners [Kenneth](https://twitter.com/KennethAlvaren5) and [Fernanda](https://twitter.com/ferji1997) from ETHLatam’s winning team, EduChain. Kenneth and Fernanda shared that they chose to hack with Waku because of their need to maintain transparent communications and feedback on their platform. Their project is blockchain-based and demands a quick and secure notification system for its users. They also shared that University officials have agreed to start trialling the EduChain platform with Masters-level scholarship students. Kudos to the EduChain team!
+
+## Improving the Store protocol 
+
+The Waku team shared three key milestones for 2024. The first is improvements to the [Store protocol](https://docs.waku.org/guides/nwaku/configure-nwaku/#configure-store-protocol-and-message-store), which provides message caching and serves it to network peers. The current version can, however, result in different nodes in the network providing you with different answers when queried. This is due to a reliability issue where the peer-to-peer network cannot absolutely guarantee that all of the messages are delivered to all of the nodes. V3 Store nodes will check with other nodes in the network to give you an accurate answer. The team is also working on incentivisation for running these nodes to provide further assurance and reliability to the network. This will require some form of payment for network users to query the nodes.
+
+## RLN v2 and v3
+
+RLN v1 is already used in the Waku network to offer message rate limiting. It is strongly recommended to use RLN because, on any libp2p network, anyone can join and spam, which can result in DoS attacks. RLN is a privacy-preserving way to limit users’ messages and give assurance against DoS attacks. We use ZK knowledge proofs for each message; relayers see the proof and validate that the users are part of the network, permitting them to send one message per second. In RLN v2 and v3, the rate limit will be configurable per user. The Waku team is now testing and running simulations of v2 to ensure it offers the scalability and reliability required. 
```

Comment:
```suggestion
RLN v1 is already used in the Waku Network to offer message rate limiting. It is strongly recommended to use RLN because, on any peer-to-peer network, anyone can join and spam, which can result in DoS attacks. RLN is a privacy-preserving way to limit users’ messages and give assurance against DoS attacks. We use ZK knowledge proofs for each message; relayers see the proof and validate that the users are part of the network, permitting them to send one message per second. In RLN v2 and v3, the rate limit will be configurable per user. The Waku team is now testing and running simulations of v2 to ensure it offers the scalability and reliability required. 
```